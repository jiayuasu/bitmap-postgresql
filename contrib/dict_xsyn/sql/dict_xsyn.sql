--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of this file.
--
SET client_min_messages = warning;
\set ECHO none
\i dict_xsyn.sql
\set ECHO all
RESET client_min_messages;

--configuration
ALTER TEXT SEARCH DICTIONARY xsyn (RULES='xsyn_sample', KEEPORIG=false);

--lexize
SELECT ts_lexize('xsyn', 'supernova');
SELECT ts_lexize('xsyn', 'grb');
