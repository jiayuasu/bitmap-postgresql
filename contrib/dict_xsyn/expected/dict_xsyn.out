--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of this file.
--
SET client_min_messages = warning;
\set ECHO none
RESET client_min_messages;
--configuration
ALTER TEXT SEARCH DICTIONARY xsyn (RULES='xsyn_sample', KEEPORIG=false);
--lexize
SELECT ts_lexize('xsyn', 'supernova');
   ts_lexize    
----------------
 {sn,sne,1987a}
(1 row)

SELECT ts_lexize('xsyn', 'grb');
 ts_lexize 
-----------
 
(1 row)

